RecoTauTag/RecoTau/interface/PFRecoTauClusterVariables.h

```diff
@@ -65,7 +65,8 @@ namespace reco {
       kPWoldDMwLT,
       kPWnewDMwLT,
       kDBoldDMwLTwGJ,
-      kDBnewDMwLTwGJ
+      kDBnewDMwLTwGJ,
+      kPhase2
```

Comment:
I propose to use more descriptive type similar to run2 ones. e.g. something like `kDBnewDMwLTwGJPhase2` instead of `kPhase2`

Reply:
kDBnewDMwLTwGJPhase2 will be used.